IO.Eventuate.Tram.Tests/docker-compose.yml

```diff
@@ -1,6 +1,16 @@
 ---
 version: '3.4'
 services:
+  dbsetup:
+    image: mcr.microsoft.com/mssql-tools@sha256:62556500522072535cb3df2bb5965333dded9be47000473e9e0f84118e248642
+    depends_on:
+    - mssql
+    environment:
+      TRAM_DB_SERVER: "LAPTOP-7G6A2VOR"
```

Comment:
See other comment about hostname for mssql

Reply:
Done